src/StatusQ/Controls/StatusPinInput.qml

```diff
@@ -0,0 +1,115 @@
+import QtQuick 2.0
+import StatusQ.Core.Theme 0.1
+
+Item {
+    id: root
+
+    property string pinInput: inputText.text
+    property var validator: inputText.validator
+    property int pinLen: 6
```

Comment:
Ah, here's the property that lets us configure the field count. Ignore the comment above!